index.js

```diff
@@ -60,6 +60,12 @@ renderPlayer(player)
 
 
 /***** Deliverable 1 *****/
+const header = document.querySelector("#header")
+
+header.addEventListener("click", function () {
+  toggleColor(header)
+})
```

Comment:
Nice!

Reply:
This won't work because toggleColor expects an element, which is different from an event. Inside the callback function in your current solution, try console.logging the `event` and the `h1Header` variables and see how they differ. toggleColor expects and only knows how to work with `h1Header`, and doesn't have any logic to handle the`event` itself